utils/src/ooinstall/install_transactions.py

```diff
@@ -126,8 +126,20 @@ def run_main_playbook(hosts, hosts_to_run_on):
         facts_env['ANSIBLE_CONFIG'] = CFG.settings['ansible_config']
     return run_ansible(main_playbook_path, inventory_file, facts_env)
 
+
 def run_ansible(playbook, inventory, env_vars):
     return subprocess.call(['ansible-playbook',
                              '--inventory-file={}'.format(inventory),
                              playbook],
                              env=env_vars)
+
+def run_uninstall_playbook():
```

Comment:
Can I get some background on this, what does install transactions mean in the installer or ansible? Not terminology I'm familiar with and we've just been working with what was there, adding new ansible calls where the old ones were located. As far as I can tell, the module is very tiny and purely focused on writing ansible inventory, and then calling ansible. Should it be renamed entirely to common / ansible.py?